virttest/utils_test/libvirt.py

```diff
@@ -3783,8 +3783,6 @@ def customize_libvirt_config(params,
             libvirtd.restart()
         obj_conf = target_conf
     else:
-        if not isinstance(config_object, utils_config.LibvirtConfigCommon):
```

Comment:
class LibvirtQemuConfig(LibvirtConfigCommon) 
Above definition should support LibvirtQemuConfig object. Could you give more reasons for the removing?

Reply:
Sorry, I rerun https://github.com/autotest/tp-libvirt/pull/4274 in new test env and found we don't need to remove this check. So close it.